snapcraft/tests/fake_servers.py

```diff
@@ -138,12 +161,8 @@ class FakePartsWikiOriginRequestHandler(BaseHTTPRequestHandler):
 
     def do_GET(self):
         logger.debug('Handling getting part origin')
-        if self.headers.get('If-None-Match') == '1111':
```

Comment:
This is only for the parts wiki origin tests not the remote parts.  I copied it over when I created these fake servers and forgot to remove that line.